libs/blocks/quiz/quizoption.js

```diff
@@ -1,40 +1,62 @@
 import { html } from '../../deps/htm-preact.js';
 
 export const OptionCard = ({
-  text, title, coverImage, cardIcon, options, disabled, selected,
+  text, title, image, icon, iconTablet, iconDesktop, options, disabled, selected, background,
 }) => {
-  const cardIconHtml = html`<div class="quiz-option-icon">
-    <img src="${cardIcon}" alt="Quiz Option Icon" />
+  const getOptionClass = () => {
+    let className = '';
+    if (icon || iconTablet || iconDesktop) className += 'has-icon ';
+    if (image) className += 'has-image ';
+    if (background) className += 'has-background ';
+    if (disabled) className += disabled;
+    if (selected) className += selected;
+    return className;
+  };
+
+  const getIconHtml = (iconUrl, iconClass) => html`<div class="quiz-option-icon ${iconClass}">
+    <img src="${iconUrl}" alt="${text || title || 'Quiz Option Icon'}" />
   </div>`;
 
-  const coverImageHtml = html`
+  const imageHtml = html`
     <div class="quiz-option-image" 
-      style="background-image: url('${coverImage}')">
+      style="background-image: url('${image}')">
```

Comment:
I'm wondering if it would be more performant and consistent with other Milo features to use srcset. 

![image](https://github.com/adobecom/milo/assets/7562106/696a588b-86e1-4f53-8c8c-0fb73794fff5)

Reply:
changed to using picture, media and srcset for icons